_data/merchants.yml

```diff
@@ -189,6 +189,8 @@
   merchants:
     - name: All For Crypto
       url: https://allforcrypto.com
+    - name: Cannabis Cup
+      url: https://shop.cannabiscup.com/
```

Comment:
I couldn't find written anywhere that they accept Monero, can you provide a link where they explicitly say they accept it?